src/Contracts/PreMigration.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace LaravelEnso\Upgrade\Contracts;
```

Comment:
@raftx24 After discussing with A we agreed that we should go with BeforeMigration instead of PreMigration as it's closer to day to day speak